tests/Feature/Profile/DefaultProfileGeneratorTest.php

```diff
@@ -62,12 +62,10 @@ class ProfileController extends Controller
 /* Auto-generated admin routes */
 Route::middleware([\'auth:\' . config(\'admin-auth.defaults.guard\'), \'admin\'])->group(static function () {
     Route::prefix(\'admin\')->namespace(\'Admin\')->name(\'admin/\')->group(static function() {
-        Route::prefix(\'admin-users\')->name(\'admin-users/\')->group(static function() {
```

Comment:
there was prefix `admin-users` before? Weird, I didn't know.

Reply:
I think @RichardDominik introduced it few commits back.